### PR TITLE
Add dev.ters.Chordcat

### PR DESCRIPTION
**Pull request by shriramters**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
Chordcat is a desktop app that helps you identify and name chords based on inputted notes.
It supports MIDI input as well as Keyboard and mouse input to input and play notes.

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:** I am the author of Chordcat. https://github.com/shriramters/chordcat

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: dev.ters.Chordcat.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/shriramters/chordcat/assets/127589779/697dc8d9-b8d6-412e-be2d-fea71f324311</image>
+      <caption>About window and main screen with midi keyboard connected and default font</caption>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="0.2.0" date="2024-09-25"/>
+  </releases>
+  <content_rating type="oars-1.0" />
```
Comment (hfiguiere) [inline dev.ters.Chordcat.metainfo.xml:67]:
```suggestion
  <content_rating type="oars-1.1" />
```

File: dev.ters.Chordcat.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <p>
+      Chordcat is a desktop app that helps you identify and name chords based on inputted notes.
+    </p>
+    <p>
+      Features:
+    </p>
+    <ul>
+      <li>
+        Displays chord names when:
+        <ul>
```
Comment (hfiguiere) [inline dev.ters.Chordcat.metainfo.xml:29]: `ul` inside `li` isn't allowed. it will fail validation.

Comment (shriramters) [reply dev.ters.Chordcat.metainfo.xml]: Thanks! I have updated that now

File: dev.ters.Chordcat.metainfo.xml
Comment (bbhtt) [inline dev.ters.Chordcat.metainfo.xml:1]: metainfo file should be here https://github.com/shriramters/chordcat/

File: dev.ters.Chordcat.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
+id: dev.ters.Chordcat
```
Comment (bbhtt) [inline dev.ters.Chordcat.yml:1]: how do I know you own ters.dev? Can you link the github somewhere on the webpage?

**Comment by bbhtt** [comment]
bot, build dev.ters.Chordcat

File: dev.ters.Chordcat.yml (hunk 3 above)
Comment (shriramters) [reply dev.ters.Chordcat.yml:1]: My name is here on https://ters.dev/

"Shriram R"

I have also added a TXT record to the dns of chordcat.ters.dev linking to the github repo

you can check it here https://www.nslookup.io/domains/chordcat.ters.dev/dns-records/txt/

or via the command line

```
shriram@think182:~$ dig @ns-cloud-e4.googledomains.com. chordcat.ters.dev. TXT

; <<>> DiG 9.18.28-1~deb12u2-Debian <<>> @ns-cloud-e4.googledomains.com. chordcat.ters.dev. TXT
; (2 servers found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 44832
;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1
;; WARNING: recursion requested but not available

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
; COOKIE: cae9b295a0b1f37701318f7ce87aec42f5862f7f68b37c0238a0659c72 (good)
;; QUESTION SECTION:
;chordcat.ters.dev.		IN	TXT

;; ANSWER SECTION:
chordcat.ters.dev.	14400	IN	TXT	"https://github.com/shriramters/chordcat"

;; Query time: 40 msec
;; SERVER: 216.239.38.110#53(ns-cloud-e4.googledomains.com.) (UDP)
;; WHEN: Thu Oct 31 17:36:18 IST 2024
;; MSG SIZE  rcvd: 131
```

Comment (bbhtt) [reply dev.ters.Chordcat.yml:1]: thanks

File: dev.ters.Chordcat.metainfo.xml
Comment (shriramters) [reply dev.ters.Chordcat.metainfo.xml:1]: Sure, I will commit it now and add it in a new release

I was following https://github.com/flathub/com.brave.Browser/blob/master/com.brave.Browser.metainfo.xml

Comment (hfiguiere) [reply dev.ters.Chordcat.metainfo.xml:1]: ah yeah following what a totally different PROPRIETARY package as an example.

Comment (shriramters) [reply dev.ters.Chordcat.metainfo.xml:1]: > ah yeah following what a totally different PROPRIETARY package as an example.

Yeah lol my bad. apologies 😅

I just fixed that in a new release 0.2.1

I have also pushed my changes to this repo

**Pull request closed without merging**